invoker/core/src/test/java/com/google/cloud/functions/invoker/GcfEventsTest.java

```diff
@@ -144,6 +145,8 @@ public void firestoreComplexData() throws IOException {
     Map<String, Object> data = cloudEventDataJson(cloudEvent);
     Map<?, ?> value = (Map<?, ?>) data.get("value");
     Map<?, ?> fields = (Map<?, ?>) value.get("fields");
+    Map<String, String> nullValues = new HashMap<String, String>();
```

Comment:
I realize you can't use `Map.of` here because of the null value, but I think you can use `Collections.singletonMap`, in which case you could just put that directly inline where the `Map.of()` was.

Reply:
I will give that a try